### DISCUSSION TITLE
Will VS Code Settings Sync be supported?

### DISCUSSION THREAD

**Discussion by paveloom** [coder/code-server#2064] [General]
In the standard client I have a [`Turn On Settings Sync`](https://code.visualstudio.com/docs/editor/settings-sync#_turning-on-settings-sync) button, in the `code-server` I can't find it. Is it because there's no stable release yet? Or is it a deliberate decision?

**Comment by code-asher** [comment]
Nope, it's not deliberate. We'll need to dig in and see if we can get it working.

**Comment by bpmct** [comment]
Is there an issue open for this?

**Reply by paveloom** [reply]
As far as I know, no. Whether it appears depends on whether this feature can be implemented.

**Comment by sr229** [comment]
Settings Sync uses an API only available for Visual Studio Codespaces and Visual Studio Code, I don't think we're allowed to use this since it violates their ToS. We should implement our own instead.
